notebooks-auth/test_cowbird_jupyter.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        raise ValueError(response_msg("\nFailed to start JupyterLab Docker container.", resp))
</pre><p>Is the error correct?</p><p>Shouldn't it be "failed to create JupyterLab user token"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>You're right, the error message is wrong. Probably forgot to change it from previous test attempts.</p>